pingcap/dm
# tests: Use log.Fatal instead of self made util function

### What problem does this PR solve? 

The behavior of `ExitWithError` is very similar to `log.Fatal`.


### What is changed and how it works?

Remove `ExitWithError` and just use `log.Fatal`.


### Check List 

Tests 

 - Manual test
    I use go run to check the output of the two affected program.
The only difference is that the log level changed from `error` to `fatal`.

Code changes


Side effects


Related changes
